src/common/sizer.cpp

```diff
@@ -2378,7 +2378,7 @@ void wxBoxSizer::RecalcSizes()
 
             wxCoord maxMajor = GetSizeInMajorDir(item->GetMaxSizeWithBorder());
 
-            // must be nonzero, fixed-size items were dealt with in previous loop
+            // must be nonzero, fixed-size items were dealt within previous loop
```

Comment:
Not sure about this one... do people really say "within a loop"? I don't think so...

Reply:
Yes, that is correct.

Reply:
Wait, maybe "with in" is correct.  Sorry, just awkward wording I guess, will change it back...